### ISSUE TITLE
Error when using DataTableRowActions component in a DataTable

### ISSUE THREAD

**Issue by girishla** [salesforce/design-system-react#1353]
I see the following in the console when the action dropdown is Clicked.

```
Warning: Failed prop type: Invalid prop `aria-disabled` of type `string` supplied to `SLDSListItem`, expected `boolean`.
    in SLDSListItem (created by SLDSList)
    in SLDSList (created by SLDSMenuDropdown)
    in div (created by SLDSDialog)
    in SLDSDialog (created by SLDSMenuDropdown)
    in div (created by SLDSMenuDropdownTrigger)
    in SLDSMenuDropdownTrigger (created by SLDSMenuDropdown)
    in SLDSMenuDropdown (created by OnClickOutside(SLDSMenuDropdown))
    in OnClickOutside(SLDSMenuDropdown) (created by SLDSDataTableRowActions)
    in td (created by SLDSDataTableRowActions)
    in SLDSDataTableRowActions (created by JobsTable)
    in tr (created by SLDSDataTableRow)
    in SLDSDataTableRow (created by SLDSDataTable)
    in tbody (created by SLDSDataTable)
    in table (created by SLDSDataTable)
    in SLDSDataTable (created by JobsTable)
    in SLDSIconSettings (created by JobsTable)
    in div (created by JobsTable)
    in JobsTable (created by JobsListView)
    in SLDSIconSettings (created by JobsListView)
    in div (created by JobsListView)
    in JobsListView (created by ZincJobs)
    in ZincJobs (created by UserIsAuthenticated(ZincJobs))
    in UserIsAuthenticated(ZincJobs) (created by Connect(UserIsAuthenticated(ZincJobs)))
    in Connect(UserIsAuthenticated(ZincJobs)) (created by withSaga(Connect(UserIsAuthenticated(ZincJobs))))
    in withSaga(Connect(UserIsAuthenticated(ZincJobs))) (created by withReducer(withSaga(Connect(UserIsAuthenticated(ZincJobs)))))
    in withReducer(withSaga(Connect(UserIsAuthenticated(ZincJobs)))) (created by Connect(withReducer(withSaga(Connect(UserIsAuthenticated(ZincJobs))))))
    in Connect(withReducer(withSaga(Connect(UserIsAuthenticated(ZincJobs))))) (created by Route)
    in Route (created by RouteWithLayout)
    in div (created by PageBase)
    in div (created by PageBase)
    in div (created by PageBase)
    in div (created by PageBase)
    in PageBase (created by RouteWithLayout)
    in CSSTransitionGroupChild (created by TransitionGroup)
    in span (created by TransitionGroup)
    in TransitionGroup (created by CSSTransitionGroup)
    in CSSTransitionGroup (created by Layout)
    in div (created by Layout)
    in div (created by Layout)
    in MuiThemeProvider (created by Layout)
    in Layout (created by WithWidth)
    in EventListener (created by WithWidth)
    in WithWidth (created by Route)
    in Route (created by withRouter(WithWidth))
    in withRouter(WithWidth) (created by Connect(withRouter(WithWidth)))
    in Connect(withRouter(WithWidth)) (created by RouteWithLayout)
    in RouteWithLayout (created by App)
    in Switch (created by App)
    in div (created by App)
    in Router (created by App)
    in App (created by Route)
    in Route (created by withRouter(App))
    in withRouter(App) (created by Connect(withRouter(App)))
    in Connect(withRouter(App))
    in Router (created by ConnectedRouter)
    in ConnectedRouter
    in Provider
```

**Comment by interactivellama** [comment]
I think this is the line to change to string. https://github.com/salesforce/design-system-react/blob/master/components/utilities/menu-list/item.jsx#L37

Re-open if you want to take it on.
